app/utils/types/datetime.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from sqlalchemy import DateTime
+from sqlalchemy.types import TypeDecorator
+
+
+class TZDateTime(TypeDecorator):
```

Comment:
Could you add comments explaining when are called these two functions and where is the class used?